pypeclub/OpenPype
# Maya: Vray fix proxies look assignment

## Fix

Fix problems with look assignment and scene with vray proxies. Fixing `Get All Assets` functionality.

There was a bug when there were vray proxies with alembics and vrmeshes combined loaded in scene, that look assigner failed during applying the look. This fix is also restoring functionality of `Get All Assets` button.

### How to test:

Load into scene vray proxy published with OpenPype - that one should be in abc format. Load some vrmeshes manually via vray menu and try to apply look to some of the loaded proxies. Look should be applied correctly. Also - `Get All Assets` button in look assigner should return all loaded assets (list was previously empty due to the bug).

Close #2231
